src/Executables/PrintMinimumGridSpacing/PrintMinimumGridSpacing.cpp

```diff
@@ -0,0 +1,154 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/program_options.hpp>
+#include <iomanip>
+
+#include "Domain/InitialElementIds.hpp"
+#include "Domain/MinimumGridSpacing.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Parallel/Printf.hpp"
+#include "Options/ParseOptions.hpp"
+
+using frame = Frame::Inertial;
```

Comment:
We generally avoid these types of global `using`s because they can cause someone else a lot of headache because all of a sudden something is defined that shouldn't be defined at that scope. Since you only use it in a couple places would you mind writing it out explicitly there?